pokemongo_bot/__init__.py

```diff
@@ -760,8 +760,18 @@ def _setup_logging(self):
         logging.getLogger("pgoapi").setLevel(log_level)
         logging.getLogger("rpc_api").setLevel(log_level)
 
-        if self.config.logging_clean and not self.config.debug:
-            formatter = Formatter(fmt='[%(asctime)s] %(message)s', datefmt='%H:%M:%S')
+        if self.config.logging:
+            logging_format = '%(message)s'
+            logging_format_options = ''
+            if ('show_log_level' in self.config.logging and self.config.logging['show_log_level']) or 'show_log_level' not in self.config.logging:
```

Comment:
OK, done and tested. All lines now follow the format you suggested as follows:

```
            if ('show_log_level' not in self.config.logging) or self.config.logging['show_log_level']:
                logging_format = '[%(levelname)s] ' + logging_format
```

I knew something wasn't quite right in there.